+adi/+AD2S1210/Rx.m

```diff
@@ -0,0 +1,129 @@
+classdef Rx < adi.common.Rx & matlabshared.libiio.base & adi.common.Attribute
+    % AD2S1210 Resolver-to-Digital Converter Class
+    % adi.AD2S1210.Rx Receives data from the AD2S1210 Resolver
+    %   The adi.AD2S1210.Rx System object is a signal source that can receive
+    %   data from the AD2S1210.
+    %
+    %   rx = adi.AD2S1210.Rx;
+    %   rx = adi.AD2S1210.Rx('uri','192.168.2.1');
+    %
+    %   <a href="https://www.analog.com/media/en/technical-documentation/data-sheets/ad2s1210.pdf">AD2S1210 Datasheet</a>
+
+    properties (Nontunable)
+        % SamplesPerFrame Samples Per Frame
+        %   Number of samples per frame, specified as an even positive
+        %   integer.
+        SamplesPerFrame = 4096
+    end
+
+    properties (Dependent)
+        % Angle
+        %   Resolver angle in Degrees.
+        Angle
+
+        % AngleScale Angular Scale
+        %   Resolver angle scale.
+        AngleScale
+
+        % Velocity Angular Velocity
+        %   Resolver velocity in revolutions per second.
+        Velocity
+
+        % VelocityScale Velocity Angular Scale
+        %   Resolver velocityscale.
+        VelocityScale
+    end
+
+    properties
+        % ExcitationFrequency Excitation Frequency
+        %   Resolver excitation frequency in Hertz.
+        ExcitationFrequency = 0;
+    end
+
+    properties (Hidden)
```

Comment:
This property is not being used - can be removed.

Reply:
Attribute was removed from the driver indeed ; removing it here too to stay in sync.

Reply:
Ah, I wasn't aware ExcitationFrequency had been removed from the driver. I was actually referring to the FramesCount property.

Reply:
It looks like the attribute is still there in the driver? https://github.com/analogdevicesinc/linux/blob/71da83b0b094a92ed4c2b8c21503638b3557ac56/drivers/staging/iio/resolver/ad2s1210.c#L609

Reply:
> Ah, I wasn't aware ExcitationFrequency had been removed from the driver. I was actually referring to the FramesCount property.

FramesCount property seems to be mandatory; here is the error message I get whent trying to remove it :

```
Abstract classes cannot be instantiated. 
Class 'adi.AD2S1210.Rx' inherits abstract methods or properties but does not implement them.
See the list of methods and properties that 'adi.AD2S1210.Rx' must implement if you do not intend the class to be abstract.

Abstract properties for class adi.AD2S1210.Rx:
    SamplesPerFrame	% defined in matlabshared.libiio.base
```

> It looks like the attribute is still there in the driver? https://github.com/analogdevicesinc/linux/blob/71da83b0b094a92ed4c2b8c21503638b3557ac56/drivers/staging/iio/resolver/ad2s1210.c#L609

There's actually an upstream proposal to convert it to a channel attribute (thus warranting the need to remove it from the Matlab code for now):
https://lore.kernel.org/all/20230929-ad2s1210-mainline-v3-17-fa4364281745@baylibre.com/